src/db/tables/featureFlags.ts

```diff
@@ -88,7 +110,11 @@ async function addFeatureFlag(flag: FeatureFlag) {
     expired_at: flag.expiredAt,
     owner: flag.owner,
   };
-  const res = await knex("feature_flags").insert(featureFlagDb).returning("*");
+  const res = await knex("feature_flags")
```

Comment:
ok just to clarify this query:
It attempts to upsert the `featureFlagDb` object, when the name existed, it updates the `deleted_at`

And the assumption here is that the `deleted_at` would be updated to null because `featureFlagDb.deleted_at` would be null? This essentially resets the feature flag from soft delete -> no longer deleted?

Reply:
Right exactly